assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -201,7 +201,12 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
         ? accessedResourceColor
         : notAccessedResourceColor
       )
+      .on('focus', (e, d) => {
+        moveBrushOnFocus(e, d.resource_name)
+        toolTip.show(e, d)
+      })
```

Comment:
Because the handlers added use `focus`, they also trigger on mouse click as well as keyboard tabbing. There's not a simple way (I've found) to determine whether it was triggered by mouse or tabbing. The result is that if users click on bars, the brush and chart may jump a little, but I've tried to make the new location good for visibility of the rest of the chart. This will need some testing.